src/applications/gi/components/profile/CalculatorForm.jsx

```diff
@@ -118,6 +119,72 @@ class CalculatorForm extends React.Component {
   renderYellowRibbon() {
     if (!this.props.displayedInputs.yellowRibbon) return null;
 
+    if (__BUILDTYPE__ !== 'production') {
+      let {
+        yellowRibbonDegreeLevelOptions,
+        yellowRibbonDivisionOptions,
+      } = this.props.inputs;
+
+      yellowRibbonDegreeLevelOptions =  yellowRibbonDegreeLevelOptions.map(value => ({ value, label: value }));
+      yellowRibbonDivisionOptions = yellowRibbonDivisionOptions.map(value => ({ value, label: value }));
+
+      return (
+        <div>
+          <RadioButtons
+            label={this.renderLearnMoreLabel({
+              text: 'Will you be a Yellow Ribbon recipient?',
+              modal: 'calcYr'
+            })}
+            name="yellowRibbonRecipient"
+            options={[
+              { value: 'yes', label: 'Yes' },
+              { value: 'no', label: 'No' }
+            ]}
+            value={this.props.inputs.yellowRibbonRecipient}
+            onChange={this.handleInputChange}/>
+          { this.props.inputs.yellowRibbonRecipient === 'yes' ?
+            <div>
+              <Dropdown
+                label="Degree Level"
+                name={'yellowRibbonDegreeLevel'}
+                alt="Degree Level"
+                options={yellowRibbonDegreeLevelOptions}
+                visible
+                value={this.props.inputs.yellowRibbonDegreeLevel}
+                onChange={this.handleInputChange}/>
+              <Dropdown
+                label="Division or school"
+                name={'yellowRibbonDivision'}
+                alt="Division or school"
+                options={yellowRibbonDivisionOptions}
+                visible
+                value={this.props.inputs.yellowRibbonDivision}
+                onChange={this.handleInputChange}/>
+              <div>
+                <label htmlFor="yellowRibbonAmount">
+                  Yellow Ribbon amount from school per year
+                </label>
+                <input
+                  type="text"
+                  name="yellowRibbonAmount"
```

Comment:
There's some accessibility stuff here I don't think I've resolved. There's nothing here that a screen reader can read but I'm not sure what I should add.

Reply:
The input needs an id, `for` attributes on labels references that instead of `name`.